lib/manager/azure-pipelines/index.ts

```diff
@@ -0,0 +1,5 @@
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)azure-pipelines?\\.ya?ml'],
```

Comment:
```suggestion
  fileMatch: ['azure.*pipelines?.*\\.ya?ml'],
```

How about this? i.e. any yml/yaml file with `azure` and `pipelines` occurring in the directory of filename?

Reply:
Thanks, I updated the regexp with your suggestion